addons/categories/widget.php

```diff
@@ -56,9 +70,6 @@ public function widget( $args, $instance ) {
 			'order'      => $instance['order'],
 		);
 
-		$icon_width  = ! empty( $instance['icon_width'] ) ? $instance['icon_width'] : 32;
```

Comment:
@mebishalnapit Why this and below lines were removed?

Reply:
@rahularyan These are now being set from here: https://github.com/anspress/anspress/pull/606/files/573228d38c65e3b8674f013d474fc2ad6029f668#diff-d9aa7d778f6a9e5c7cb06e952a0a2d4e59723416eb6f51ac7fdc93ef0aa81a97R40-R53 and hence not needed anymore so they were deleted.

Reply:
You are setting default value there. What if those args are set as an empty string or 0?

Reply:
Maybe adding those values as default might be better option to display the icons. If not then, I'll remove them and make those two value to be set as needed, i.e., same as previous.